orchestrator/gravity_utils/Cargo.toml

```diff
@@ -25,9 +24,6 @@ num-bigint = "0.4"
 log = "0.4"
 url = "2"
 sha3 = "0.9"
-tiny-bip39 = "0.8.0"
-bitcoin = { version = "=0.27", features = ["use-serde"] }
```

Comment:
I'm confused as to how these can be removed considering I'm fairly certain they are used (perhaps at the very least in the CLI?). I wonder if they are included in the global workspace through other Cargo.toml files incidentally. If so, it working transiently is more an artifact than a desired outcome -- we want each workspace crate to be explicit about its dependencies.

Reply:
Unless these really are declared in gravity_utils and never imported...I'd have to look through the whole crate to verify.

Reply:
I only deleted those that were imported and never used expect for openssl stuff. It looked like a bunch were copy/pasted between crates.